kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportPage.vue

```diff
@@ -4,99 +4,101 @@
+          <CoreTable :emptyMessage="coachString('lessonListEmptyState')">
+            <template #headers>
+              <th>{{ coachString('titleLabel') }}</th>
+              <th>{{ coreString('progressLabel') }}</th>
+            </template>
+            <template #tbody>
+              <transition-group
+                tag="tbody"
+                name="list"
+              >
+                <tr
+                  v-for="tableRow in lessonsTable"
+                  :key="tableRow.id"
                 >
-                  <tr
-                    v-for="tableRow in lessonsTable"
-                    :key="tableRow.id"
-                  >
-                    <td>
-                      <KRouterLink
-                        :to="
-                          classRoute('ReportsLearnerReportLessonPage', {
-                            lessonId: tableRow.id,
-                          })
-                        "
-                        :text="tableRow.title"
-                        icon="lesson"
-                      />
-                    </td>
-                    <td>
-                      <StatusSimple :status="tableRow.status" />
-                    </td>
-                  </tr>
-                </transition-group>
-              </template>
-            </CoreTable>
-          </KGridItem>
-          <KGridItem :layout12="{ span: $isPrint ? 12 : 6 }">
-            <h2>{{ coachString('quizzesAssignedLabel') }}</h2>
-            <CoreTable
-              :class="{ print: $isPrint }"
-              :emptyMessage="coachString('quizListEmptyState')"
-            >
-              <template #headers>
-                <th>{{ coachString('titleLabel') }}</th>
-                <th>{{ coreString('progressLabel') }}</th>
-                <th>{{ coreString('scoreLabel') }}</th>
-              </template>
-              <template #tbody>
-                <transition-group
-                  tag="tbody"
-                  name="list"
+                  <td>
+                    <KRouterLink
+                      :to="
+                        classRoute('ReportsLearnerReportLessonPage', {
+                          lessonId: tableRow.id,
+                        })
+                      "
+                      :text="tableRow.title"
+                      icon="lesson"
+                    />
+                  </td>
+                  <td>
+                    <StatusSimple :status="tableRow.status" />
+                  </td>
+                </tr>
+              </transition-group>
+            </template>
+          </CoreTable>
+          <KButton
+            v-if="showViewMoreButton"
+            :text="coreString('viewMoreAction')"
+            appearance="raised-button"
+            @click="loadMoreLessonTable()"
+          />
+        </KPageContainer>
+      </KGridItem>
+
+      <KGridItem
+        :layout12="{ span: 6 }"
+        :layout8="{ span: 4 }"
+      >
+        <KPageContainer class="content-spacing right-container">
```

Comment:
`left-container` and `right-container` are the same. I think we can consolidate it into 1 class? Might even be better to apply one class that sets the `padding`, `height` and `width`.